sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/tests/SessionRecords/ObjectAnchorsConversionClientLiveTests/ObserveExistingAssetConversion.json

```diff
@@ -1,86 +1,77 @@
-        ],
-        "x-mrc-cv": "jnXja04WBUSlCloKQx8arg",
-        "x-ms-client-request-id": "ffb54b3f090a623da467d1b237b5e1d1",
+        "traceparent": "00-a172c3d21f267449898f5bb0a3b3002d-2fc030643487ca45-00",
+        "User-Agent": "azsdk-net-MixedReality.ObjectAnchors.Conversion/0.1.0-alpha.20210325.1 (.NET Framework 4.8.4300.0; Microsoft Windows 10.0.19042 )",
+        "x-mrc-cv": "zBI5Ts2JLESJGqOBvvHIWA",
+        "x-ms-client-request-id": "34da8de384f4b2b0e8a675efe9a522ff",
         "x-ms-return-client-request-id": "true"
       },
       "RequestBody": null,
       "StatusCode": 200,
       "ResponseHeaders": {
         "Connection": "keep-alive",
-        "Content-Length": "243",
+        "Content-Length": "247",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Mon, 22 Feb 2021 23:33:58 GMT",
-        "MS-CV": "Q2srO4IlHk2aMwG8WL3Okg.0",
+        "Date": "Thu, 25 Mar 2021 20:15:47 GMT",
+        "MS-CV": "bDRsd8WveESgJSebkT7D3A.0",
         "X-Content-Type-Options": "nosniff"
       },
       "ResponseBody": {
-        "inputAssetUri": "https://aoaprodstoreeu2.blob.core.windows.net/6b3ce8f5-fe52-47f1-ae7a-f6e7513ee09b/dbc04871-6333-4b81-9301-6956a67fb1fc?sv=2018-03-28\u0026sr=b\u0026sig=DxQmMqz1gMYSRjB2Aavms9QvLEo53t54LIGbLgVhWmM%3D\u0026se=2021-02-23T00%3A33%3A58Z\u0026sp=cw"
+        "inputAssetUri": "https://aoaprodstoreeu2.blob.core.windows.net/f2adfdb1-0da3-4e3e-a097-4355911dea83/a245e194-28e5-42c8-ae9d-fc56fcf56c59?sv=2018-03-28\u0026sr=b\u0026sig=Oylc8KFKF9G73mobi8aYX%2B7IDLCGm%2Bf6069EwQ000iM%3D\u0026se=2021-03-25T21%3A15%3A47Z\u0026sp=cw"
       }
     },
     {
-      "RequestUri": "https://aoaprodstoreeu2.blob.core.windows.net/6b3ce8f5-fe52-47f1-ae7a-f6e7513ee09b/dbc04871-6333-4b81-9301-6956a67fb1fc?sv=2018-03-28\u0026sr=b\u0026sig=DxQmMqz1gMYSRjB2Aavms9QvLEo53t54LIGbLgVhWmM%3D\u0026se=2021-02-23T00%3A33%3A58Z\u0026sp=cw",
+      "RequestUri": "https://aoaprodstoreeu2.blob.core.windows.net/f2adfdb1-0da3-4e3e-a097-4355911dea83/a245e194-28e5-42c8-ae9d-fc56fcf56c59?sv=2018-03-28\u0026sr=b\u0026sig=Oylc8KFKF9G73mobi8aYX%2B7IDLCGm%2Bf6069EwQ000iM%3D\u0026se=2021-03-25T21%3A15%3A47Z\u0026sp=cw",
       "RequestMethod": "PUT",
       "RequestHeaders": {
         "Content-Length": "215369",
         "If-None-Match": "*",
-        "User-Agent": [
-          "azsdk-net-Storage.Blobs/12.6.0",
-          "(.NET Core 4.6.29719.03; Microsoft Windows 10.0.19041 )"
-        ],
+        "traceparent": "00-e7e00ce6a7e8784396a9283b56a59caf-85f0f6e59290b14f-00",
+        "User-Agent": "azsdk-net-Storage.Blobs/12.8.0 (.NET Framework 4.8.4300.0; Microsoft Windows 10.0.19042 )",
         "x-ms-blob-type": "BlockBlob",
-        "x-ms-client-request-id": "d7cac6e362dec65e30dbbf445b977659",
+        "x-ms-client-request-id": "689e964aa814c8ae3d226157b6b2fd27",
         "x-ms-return-client-request-id": "true",
-        "x-ms-version": "2019-12-12"
+        "x-ms-version": "2020-04-08"
```

Comment:
FYI @pakrym and @jastenze when updating the version of the `Azure.Storage.Blobs` package to `12.8` these tests started failing because of the mismatch in the recording on the parameter `x-ms-version`.

Reply:
We might want to pin their client to a specific API version in https://github.com/Azure/azure-sdk-for-net/blob/0171e46e48d260b98c5c1d575aea1ae305b034f0/sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/tests/ObjectAnchorsConversionClientLiveTests.cs 

So instead of `new BlobClientOptions()` we do `new BlobClientOptions(ServiceVersion.2019-12-12)` so they are not broken every time we update storage.